dist/angular-dynamic-layout.js

```diff
@@ -153,20 +160,24 @@
 
   angular
     .module('dynamicLayout')
-    .directive('layoutOnLoad', ['$rootScope', layoutOnLoad]);
+    .directive('layoutOnLoad', ['$rootScope', '$timeout', layoutOnLoad]);
 
   /*
    * Directive on images to layout after each load
    */
-  function layoutOnLoad($rootScope) {
+  function layoutOnLoad($rootScope, $timeout) {
 
     return {
-        restrict: 'A',
-        link: function(scope, element) {
-          element.bind('load error', function() {
-            $rootScope.$broadcast('layout');
+      restrict: 'A',
+      link: function(scope, element) {
+        element.bind('load error', function() {
+          var timeoutId;
```

Comment:
Did you get a chance to make this small change? would be happy to merge the PR after that